apps/wallet-mobile/src/yoroi-wallets/cardano/utxoManager/utxos.ts

```diff
@@ -13,13 +13,29 @@ export const collateralConfig: CollateralConfig = {
 export function isPureUtxo(utxo: RawUtxo) {
   return utxo.assets.length === 0
 }
+
+export const hasValue = (utxo: RawUtxo) => {
+  return new BigNumber(asQuantity(utxo.amount)).gte(0)
+}
+
 export function isAmountInCollateralRange(amount: RawUtxo['amount'], {maxLovelace, minLovelace}: CollateralConfig) {
   const value = new BigNumber(asQuantity(amount))
   const min = new BigNumber(minLovelace)
   const max = new BigNumber(maxLovelace)
   return value.gte(min) && value.lte(max)
 }
 
+export const findCollateralCandidates = (
```

Comment:
shouldn't also receive a limit (3) of records to return ? when returning for CIP30?

Reply:
yap, please ping @vsubhuman